pass down $LDFLAGS to make sure libraries (in particular libpng) is picked up from the right location
author: Miguel Dias Costa (National University of Singapore)
--- makeinclude.in.orig	2018-03-21 10:00:51.806724215 +0100
+++ makeinclude.in	2018-03-21 10:01:19.017232991 +0100
@@ -93,7 +93,7 @@
 CAIROFLAGS	= @CAIROFLAGS@
 DSOFLAGS	= -L. @DSOFLAGS@
 LDFLAGS		= $(OPTIM) @LDFLAGS@
-LDLIBS		= @LIBS@
+LDLIBS		= @LDFLAGS@ @LIBS@
 GLDLIBS		= @GLLIB@ @LIBS@
 LINKFLTK	= @LINKFLTK@
 LINKFLTKGL	= @LINKFLTKGL@
